Traceback (most recent call last):
  File "main.py", line 88, in <module>
    main()
  File "main.py", line 84, in main
    model_trained = train(dataset, config,  device)
  File "main.py", line 48, in train
    loss, timing_loss, choice_loss  = model(arr_b.float(), arr_c.float(), arr_delta_time.float(), event_data, non_event_data, estimate_length, choice_data_dict)
  File "/home/dalab5/miniconda3/envs/GNN/lib/python3.8/site-packages/torch/nn/modules/module.py", line 889, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/home/dalab5/Projects/MA_packed/model.py", line 180, in forward
    non_event_lambdas = self.timing_net(mat_b, mat_c,  non_event_data) # (L_Neg, )
  File "/home/dalab5/miniconda3/envs/GNN/lib/python3.8/site-packages/torch/nn/modules/module.py", line 889, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/home/dalab5/Projects/MA_packed/model.py", line 240, in forward
    print(rate.size())
RuntimeError: mat1 and mat2 shapes cannot be multiplied (4x16 and 32x1)
CUDA availability: True
torch.Size([1, 21, 32])
torch.Size([1, 4, 32]) torch.Size([32, 1])